### PR TITLE
Add jacoco maven plugin

### PR DESCRIPTION
**Pull request by jhorstmann**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -184,6 +184,52 @@
...
+                        </goals>
+                    </execution>
+                    <execution>
+                        <id>report</id>
+                        <goals>
+                            <goal>report</goal>
+                        </goals>
+                    </execution>
+                    <!--
+                    <execution>
```
Comment (tmuehl) [inline pom.xml]: Should this stay?

**Comment by tmuehl** [comment]
👍

File: pom.xml (hunk 1 above)
Comment (jhorstmann) [inline pom.xml]: > Should this stay?

I wanted to avoid having to look up the syntax once we actually have enough coverage to start enforcing it.

**Comment by jhorstmann** [comment]
> :+1:

Looks like you are not part of the zalando or zalando-incubator org and thus zappr does not let you approve the PR.

**Comment by lukasniemeier-zalando** [comment]
:+1:

**Comment by tmuehl** [comment]
> Looks like you are not part of the zalando or zalando-incubator org and thus zappr does not let you approve the PR.

Hmm, actually I am. However, my membership was private, wondering if this is an issue. I changed it. Thanks for pointing out!

**Pull request merged**